blackbox/docs/admin/ingestion/sources/event_hub.rst

```diff
@@ -0,0 +1,189 @@
+.. highlight:: psql
+.. _ingest_event_hub:
+
+================================
+Azure Event Hub Ingestion Source
+================================
+
+.. sidebar:: Overview
+
+   .. figure:: event-hub-03.png
+      :alt: A diagram of CrateDB cluster acting a consumer of an Azure Event Hub.
+
+`Azure Event Hub`_ is a telemetry ingestion service that can collect and store
+millions of events, from services such as the `Azure IoT Hub`_.
+
+The Azure Event Hub ingestion source allows CrateDB to function as a consumer
+of events that are published to a hub, ingesting the data into user defined
+tables. These ingestion pipeline can be defined according to
+:ref:`user defined ingestion rules <administration-ingestion-rules>`. Once
+ingested, this data can be queried, aggregated and transformed by a CrateDB
+client.
+
+Using CrateDB as an Azure Event Hub consumer allows you to consume IoT events
+in real time as they are published, without the need for an intermediary
+process to collect and forward the data.
+
+.. NOTE::
+
+   The Azure Event Hub ingestion source is an
+   :ref:`enterprise feature <enterprise_features>`.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+.. _ingest_event_hub_config:
+
+Configuration
+=============
+
+This ingestion source adds a few additional settings that you can configure.
+
+These node settings can be :ref:`configured like usual <config>`, via the
+``crate.yml`` configuration file or as command line parameters using the ``-C``
+option.
+
+Node Settings
+-------------
+
+.. NOTE::
+
+   Node settings only affect the node they are configured on.
+
+**ingestion.iot_hub.enabled**
```

Comment:
wondering, if we should keep configuration names as `iot_hub` or change it to `event_hub` as well, since these seem to be all related to event_hub.. ?

Reply:
I'm going to change them to `event_hub` - makes more sense